scripts/exregional_make_ics.sh

```diff
@@ -643,9 +723,9 @@ POST_STEP
 #
 #-----------------------------------------------------------------------
 #
-if [ "${CPL_AQM}" = "TRUE" ]; then
+if [ $(boolify "${CPL_AQM}") = "TRUE" ]; then
   COMOUT="${COMROOT}/${NET}/${model_ver}/${RUN}.${PDY}/${cyc}${SLASH_ENSMEM_SUBDIR}" #temporary path, should be removed later
-  if [ "${COLDSTART}" = "TRUE" ] && [ "${PDY}${cyc}" = "${DATE_FIRST_CYCL:0:10}" ]; then
+  if [ $(boolify "${COLDSTART}") = "TRUE" ] && [ "${PDY}${cyc}" = "${DATE_FIRST_CYCL:0:10}" ]; then
```

Comment:
Due to the transition of `var_defns.sh` to `var_defns.yaml`, the nature of `DATE_FIRST_CYCL` has changed.  Instead of `YYYYMMDDHH`, it is now `YYYY-MM-DD HH:00:00`.  This is causing the `[ "${PDY}${cyc}" = "${DATE_FIRST_CYCL:0:10}" ]` section to fail, causing the AQM test to fail for `COLDSTART` tests.  Is it possible to make the yaml file reproduce the bash file for `DATE_FIRST_CYCL`?

Reply:
I will see what I can do!

Reply:
@MichaelLueken I just pushed a change that feels a little "hacky" to me, but should do the trick. Let me know what think about the code. Do you mind re-running it with the failed case you were seeing if you approve?

Reply:
The rerun of the failed AQM cold start case is now passing with the modification that you pushed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
aqm_grid_AQM_NA13km_suite_GFS_v16_20240718123320                   COMPLETE            6325.57
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE            6325.57
```
Is there a reason that the transition from bash to YAML caused the `DATE_*_CYCL` variables to be treated differently?  Is this expected behavior, or could this be pointing to an issue with the implementation?

I'm running one last warm start test to make sure everything works correctly with that configuration.